TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -461,12 +461,9 @@ static inline void reDrawProgress(uint8_t prevProgress)
 {
   uint8_t nextProgress = getPrintProgress();
 
-  if (nextProgress != prevProgress)
-  { // we need speed, do not draw anything if progress isn't changed
-    if (nextProgress > prevProgress)
-      reDrawProgressBar(prevProgress, nextProgress, PB_FILL, PB_STRIPE_ELAPSED);
-    else  // if regress, swap indexes and colors
```

Comment:
that case was present just at print start before M73 is intercepted. In that case progress based on file size is used and if the file size is small the initial percentage can be significative (e.g. 21%). Then with M73 the progress is properly set (e.g. 10%). 

try this example:

[Cura_sample.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/8103736/Cura_sample.zip)

Reply:
The example is an unrealistic one as Cura puts M73 P0 right in the second line of the gcode file.

```
;FLAVOR:Marlin
M73 P0
M117 Time Left 0h36m11s
;TIME:2171
;Filament used: 2.31565m
;Layer height: 0.2
;MINX:136.95
;MINY:136.95
;MINZ:0.2
;MAXX:173.05
;MAXY:173.05
;MAXZ:25.2
;POSTPROCESSED
;ARCWELDERPROCESSED
;ZOFFSETPROCESSED
; Postprocessed by [ArcWelder](https://github.com/FormerLurker/ArcWelderLib)
; Copyright(C) 2020 - Brad Hochgesang
; resolution=0.05mm
; path_tolerance=5%
; max_radius=1000000.00mm
; g90_influences_extruder=True
; allow_dynamic_precision=True
; default_xyz_precision=3
; default_e_precision=5

;Generated with Cura_SteamEngine 4.13.1
```